doc/source/developer_guide/dg_adding_diagnostics.rst

```diff
@@ -0,0 +1,57 @@
+:tocdepth: 3 
+
+.. _adddiag:
+
+Adding diagnostics
+==================
+
+Icepack only produces ASCII (text) log output for four points (full model with ITD, an initially ice free point, a land point, and a case with a slab ocean). Each of these files contains the state information for that point. Sometimes additional variables are required in this output. The procedure for adding diagnostic variables is outlined here.
+
+#. For non-BGC variables, one should edit **icedrv\_diagnostics.F90**:
+
+   -  If the variable is already defined within the code, then add it to a "use" statement in the subroutine
+      ``runtime_diags``.
+
+   -  If the variable is just a scalar, then follow the example of "aice". Copy the write statement for Qa to
+      a place in the output list where it is most appropriate. The format "900" is appropriate for most scalars.
+      Edit the copied statement to be the variable you want. The following example adds snow-melt (melts).
```

Comment:
Here is would add what n is, e.g.

> The following example adds snow melt (melts) for each of the n points.

(no hyphen needed)

Reply:
I've added a bit more here. Hopefully this is helpful.